### DISCUSSION TITLE
作者您好，可否在后期版本中考虑增加忽略某种颜色的识别，或者仅识别某种颜色？

### DISCUSSION THREAD

**Discussion by zhuyuan918** [hiroi-sora/Umi-OCR#662] [Q&A]
因为在批量识图的过程中，有些图片的位置会存在些许出入，这直接导致现有的忽略区域功能无效，另外有些时候只需要提取特定内容，所以指定颜色确实很重要，望考虑，谢谢

**Comment by hiroi-sora** [comment]
此功能看起来比较小众，开发优先级不高。或许你可以先用图像处理软件对素材做预处理（比如二值化），再导入OCR。
